src/__experimental__/components/checkbox/checkbox.style.js

```diff
@@ -0,0 +1,204 @@
+import styled, { css } from 'styled-components';
+import PropTypes from 'prop-types';
+import baseTheme from '../../../style/themes/base';
+import FieldHelpStyle from '../field-help/field-help.style';
+import LabelStyle from '../label/label.style';
+import StyledHelp from '../help/help.style';
+import { THEMES } from '../../../style/themes';
+
+const StyledCheckbox = styled.div`
+  padding-top: 8px;
+
+  input[type=checkbox] {
+    cursor: pointer;
+    opacity: 0;
+    margin: 0;
+    z-index: 999;
+  }
+
+  svg {
+    background-color: #ffffff;
+    border: solid 1px #668592;
+  }
+
+  input[type=checkbox],
+  svg {
+    position: absolute;
+  }
+
+  .carbon-checkbox__input,
+  input[type=checkbox],
+  svg {
+    box-sizing: border-box;
+    display: inline-block;
+    height: 16px;
+    padding: 1px;
+    vertical-align: middle;
+    width: 16px;
+  }
+
+  input[type=checkbox]:not([disabled]) {
+    &:focus + svg,
+    &:hover + svg {
+      outline: solid 2px #ffb403;
```

Comment:
There'll be a variable for the color. Also, is it 3px?

Reply:
Updated